psychopy/data.py

```diff
@@ -663,8 +667,6 @@ def saveAsExcel(self, fileName, sheetName='rawData',
             wb.properties.creator = 'PsychoPy' + psychopy.__version__
             newWorkbook = True
 
-        ew = ExcelWriter(workbook=wb)
```

Comment:
Fairly sure. It worked with openpyxl 2.3.2, which is what I had before - I think workbook.save() has been in the package for a while.